lib/mongoose-core.module.ts

```diff
@@ -39,6 +39,8 @@ export class MongooseCoreModule {
       ...mongooseOptions
     } = options;
 
+    mongoose.set('useCreateIndex', true);
```

Comment:
useCreateIndex must be an optional settings

Reply:
@nartc why is it required?

Reply:
It is a deprecation warning on my end. This is a hasty yet careless PR. I'll make change as soon as possible. https://mongoosejs.com/docs/deprecations.html